layers/meta-opentrons/recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -60,16 +64,26 @@ MAX_SYSTEMFS_SIZE = "1536"
 # create the opentrons ot3 manifest (VERSION.json) file
 python do_create_opentrons_manifest() {
     bb.note("Create the manifest json for for ot3-system.zip")
+    import subprocess
     import json
     import os
 
+    # Get the oe-core version, sha, branch
+    try:
+        oe_version = subprocess.check_output(['git', 'describe', '--tags', '--always']).decode().strip()
+        oe_sha = subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode().strip()
+        oe_branch = subprocess.check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD']).decode().strip()
+    except subprocess.CalledProcessError as cpe:
+        bb.error("Could not get oe-core version - %s" % cpe)
+        exit()
 
+    # Create the manifest dictionary
     opentrons_manifest = {
         'robot_type': d.getVar('ROBOT_TYPE'),
-        'build_type': os.getenv('OT_BUILD_TYPE', 'unknown/dev'),
-        'openembedded_version': d.getVar('version', 'unknown'),
-        'openembedded_sha': d.getVar('version', 'unknown'),
-        'openembedded_branch': d.getVar('version', 'unknown')
+        'build_type': d.getVar('OT_BUILD_TYPE', 'develop'),
```

Comment:
we're using develop here as a default but the common values are `debug` and `release`. I like `develop` better than `debug` - can we use that as an intended value for `OT_BUILD_TYPE` also?